package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fury-adapter-swagger",
-  "version": "0.12.1",
+  "version": "0.12.2",
```

Comment:
Please make this a minor version bump, the reason being is that it is a breaking change that has a different peerDependency on Fury.